test/CMakeLists.txt

```diff
@@ -105,11 +112,16 @@ foreach(_test Callbacks Degenerate ManufacturedSolution ComparisonWithBoost)
     file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BVH_${_bounding_volume}.cpp.tmp"
       "#include <ArborX_LinearBVH.hpp>\n"
       "#include <ArborX_KDOP.hpp>\n"
+      "#include \"Search_UnitTestHelpers.hpp\"\n"
       "using KDOP6 = ArborX::Experimental::KDOP<6>;\n"
       "using KDOP14 = ArborX::Experimental::KDOP<14>;\n"
       "using KDOP18 = ArborX::Experimental::KDOP<18>;\n"
       "using KDOP26 = ArborX::Experimental::KDOP<26>;\n"
-      "template <class MemorySpace> using ArborX__BoundingVolumeHierarchy_${_bounding_volume} = ArborX::BasicBoundingVolumeHierarchy<MemorySpace, ArborX::Details::PairIndexVolume<${_bounding_volume}>, ArborX::Details::DefaultIndexableGetter, ${_bounding_volume}>;\n"
+      "template <class MemorySpace>\n"
+      "using ArborX__BoundingVolumeHierarchy_${_bounding_volume} =\n"
```

Comment:
We only want to test the legacy code path here?

Reply:
Yes. `BVH` only has `MemorySpace`, no bounding volume, so impossible to run KDOPs with.